### ISSUE TITLE
Only offer email fields for 'author's email field'?

### ISSUE THREAD

**Issue by mjl** [feincms/form-designer#39]
How about this? Not sure whether it's in scope for form-designer or people are expected to roll their own solution?

```python
def email_field_choices(form: forms.ModelForm | None, required: bool=True) -> list[tuple[str, str]]:
    if not form:
        return []
    email_fields = form.instance.fields.filter(type='email')
    choices = [] if required else [('', '--')]
    choices.extend([(_.name, _.title) for _ in email_fields])
    return choices

...

class Form(models.Model):
    CONFIG_OPTIONS = [
        ...
                        "author_email_field",
                        forms.ChoiceField(
                            required=False,
                            choices=email_field_choices(form, required=False)
```

**Comment by mjl** [comment]
merged #40

**Comment by matthiask** [comment]
Thanks!
